docs/stdlib/json.rst

```diff
@@ -458,10 +456,11 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 .. eql:function:: std::json_object_unpack(json: json) -> \
                   set of tuple<str, json>
 
-    Return set of key/value tuples that make up the JSON object.
+    Returns a set of key/value tuples that make up a :eql:type:`json`
+    object.
```

Comment:
```suggestion
    Returns the data in a :eql:type:`json` object as a set of key/value tuples.
```
- Better reflects the data is being transformed before being returned
- Reduces back to one line